tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/TestReportTest.java

```diff
@@ -46,15 +46,20 @@
     private Path resultsJsonPath;
     private JsonObject resultObj;
 
+    private static final int TOTAL_TESTS = 0;
+    private static final int PASSED = 1;
+    private static final int SKIPPED = 2;
+    private static final int FAILED = 3;
+
     @BeforeClass
     public void setup() throws BallerinaTestException {
         balClient = new BMainInstance(balServer);
         projectPath = projectBasedTestsPath.resolve("test-report-tests").toString();
-        resultsJsonPath = projectBasedTestsPath.resolve("test-report-tests").resolve("target").resolve("report")
-                .resolve("test_results.json");
+        resultsJsonPath = projectBasedTestsPath.resolve("test-report-tests").resolve("target").resolve(
```

Comment:
Break the new line from the start of `.resolve(`

Reply:
Corrected formatting